docker/grouper/Dockerfile

```diff
@@ -0,0 +1,132 @@
+FROM jeanblanchard/java:jdk-7
```

Comment:
We need to have a JDK to install Grouper and Grouper does complain (although it seems to work fine) if it runs against Java 8.